tests/test_train.py

```diff
@@ -101,3 +103,47 @@ def test_parallel_train(tmp_path):
 
     return_code = subprocess.call(["python", "train.py"] + args)
     _assert_eq(return_code, 0)
+
+
+def test_multiple_workers(tmp_path):
```

Comment:
could you skip that one if redis is not installed? (that would be the case locally or in docker)